deformable_ompl/ComputeDVXLCostQuery[] queries
---
deformable_ompl/ComputeDVXLCostResult[] results
